documentation/installation.md

```diff
@@ -46,14 +46,26 @@ In short, for a quick development roundtrip:
    1. Run `mvn clean install` in the `studio-server` folder.
 
       Checkpoint: A zip file exists in `studio-server/target`. 
-   2. Run `pnpm install && pnpm -r run build && pnpm -r run package` in the folder `studio-client`.
+   2. Run `npm install -g pnpm@8.6 && pnpm install && pnpm -r run build && pnpm -r run package` in the folder `studio-client`.
   
       Checkpoint: A zip file exists in `studio-client/apps/main/content-hub-adapter-rss/build`.
 3. Create a directory for studio-server plugins, e.g. `/tmp/studio-server-plugins`,
    and copy `content-hub-adapter-rss/studio-server/target/studio-server.content-hub-adapter-rss-<version>.zip`
    into that directory.
-4. Start the studio server as usual, e.g. `mvn spring-boot:run`, with an additional property `-Dplugins.directory=/tmp/studio-server-plugins`
-5. Start the studio client with an additional property `-DadditionalPackagesDirs=/.../content-hub-adapter-rss/studio-client/target/app`
+4. Start the studio server as usual, e.g. `mvn spring-boot:run`, with an additional property `-Dplugins.directories=/tmp/studio-server-plugins`
+5. In your Blueprint studio-client workspace, there is a file `apps/studio-client/apps/main/app/jangaroo.config.js`,
+   which contains a structure like
+   ```
+   module.exports = jangarooConfig({
+     additionalPackagesDirs: [
+       "./build/additional-packages",
+     ],
+     ...
+   });
+   ```
+   Add the absolute path of the `content-hub-adapter-rss/studio-client/apps/main/content-hub-adapter-rss` directory
```

Comment:
Won't that cause a clash as the default Blueprint Studio Client build will already download the default RSS plugin? See `apps/studio-client/apps/main/app/plugins.json`.

Reply:
A note like you propose (_"If the plugin is already bundled in your Blueprint version..."_) would surely help. In case people want to adjust the plugin and replace the bundled one, they could be made aware of the `plugins.json` to check for and remove an already bundled version.